pepper-apis/src/main/java/org/broadinstitute/ddp/pex/TreeWalkInterpreter.java

```diff
@@ -365,11 +368,12 @@ private Object applyFormInstancePredicate(InterpreterContext ictx, PexParser.For
         }
     }
 
-    private boolean evalQuestionQuery(InterpreterContext ictx, QuestionQueryContext ctx) {
+    private Object evalQuestionQuery(InterpreterContext ictx, QuestionQueryContext ctx) {
         String userGuid = getUserGuidByUserType(ictx, ctx.USER_TYPE());
         String studyGuid = extractString(ctx.study().STR());
         String activityCode = extractString(ctx.form().STR());
         String stableId = extractString(ctx.question().STR());
+        StudyDto studyDto = new JdbiUmbrellaStudyCached(ictx.getHandle()).findByStudyGuid(studyGuid);
```

Comment:
At this point in the code, we have not checked whether the guids/codes/ids are valid. Someone might accidentally put in a wrong study guid. We should check that the study exists, just like how we're checking the activity exists on L378 below.

Reply:
Fixed